### PR TITLE
Added new LeftRightIconButton component

### PR DESCRIPTION
**Pull request by gullwings13**
### PR Checklist:

Hey there! It's great to have you here. Here's a quick checklist to help you make a shiny PR:

* [x] Have you checked out the guidelines in our [Contributing](https://github.com/FightPandemics/FightPandemics/blob/master/CONTRIBUTING.md) document?
* [x] Have you looked if there might be other open [Pull Requests](https://github.com/FightPandemics/FightPandemics/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) for the same update/change?

### Quick Description of the PR

Created a new button component the relies on submit button to meet the requirements of https://github.com/FightPandemics/FightPandemics/issues/294. This could refactored to rely on base button if preffered. It believe it meets the requirements of the issue.

### An Overview of the Changes
I made some assumptions in regards to the function and design of the buttons:
- It only accepts a single icon and in some cases I think the background color will make the icon disappear on hover. I noticed this is the case for other buttons on the site, so assuming its ok here too.
- I have hard coded the width to be 192px, should this be more fluid?
- There were some other buttons which appear to deal with icons in the repo, I assumed that those are not suitable for the task

Request:
Screen Shot 2020-05-10 at 9 45 47 PM
Example usage:
icon button examples

Example usage code:
```
<div>
	<LeftRightIconButton
	  tertiary={true}
	  inline={true}
	  style={{ fontSize: "20px" }}
	  disabled={true}
	  icon={<SvgIcon src={leftArrow} />}
	>
	  	Back
  	</LeftRightIconButton>
	<LeftRightIconButton
	  primary={true}
	  inline={true}
	  righticon={true}
	  disabled={true}
	  icon={<SvgIcon src={rightArrow} />}
	>
	  	Next
  	</LeftRightIconButton>
</div>
<div>
	<LeftRightIconButton
	  tertiary={true}
	  inline={true}
	  style={{ fontSize: "20px" }}
	  icon={<SvgIcon src={leftArrow} />}
	>
	  	Back
	</LeftRightIconButton>
	<LeftRightIconButton
	  primary={true}
	  inline={true}
	  righticon={true}
	  icon={<SvgIcon src={rightArrow} />}
	>
  		Next
	</LeftRightIconButton>
</div>
<div>
	<LeftRightIconButton
	  secondary={true}
	  inline={true}
	  style={{ fontSize: "20px" }}
	  icon={<SvgIcon src={leftArrow} />}
	>
  		Back
	</LeftRightIconButton>
	<LeftRightIconButton
	  primarylight={true}
	  inline={true}
	  righticon={true}
	  icon={<SvgIcon src={rightArrow} />}
	>
  		Next
	</LeftRightIconButton>
</div>
```

### CODE REVIEW

File: client/src/components/Button/LeftRightIconButton.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      }
+    `}
+
+  ${(props) =>
+    props.disabled &&
+    css`
+      &.am-button > .am-button-icon {
+        opacity: 0.6;
+      }
+      &.am-button.am-button-disabled {
```
[the remark this answers is not in the document]
Comment (gullwings13) [reply client/src/components/Button/LeftRightIconButton.js:42]: Sure thing. You are referring to the code that is dealing with the disabled styles?

**Comment by gullwings13** [comment]
I've moved the opacity for the disabled button icon into the base button component. I think the color of the disabled button text could possibly be moved into the SubmitButton component potentially (this LeftRightIconButton is extending the SubmitButton). Let me know if you have a preference for where it is. Thanks!

File: client/src/components/Button/LeftRightIconButton.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (anniegiang) [reply client/src/components/Button/LeftRightIconButton.js:39]: Yup!

**Comment by anniegiang** [comment]
> I've moved the opacity for the disabled button icon into the base button component. I think the color of the disabled button text could possibly be moved into the SubmitButton component potentially (this LeftRightIconButton is extending the SubmitButton). Let me know if you have a preference for where it is. Thanks!

Sounds good! If the styles will be shared across button I think it makes sense to put it in `BaseButton`.

**Comment by gullwings13** [comment]
Should then the theme colors in the submit button also be moved to base button?

**Comment by anniegiang** [comment]
> Should then the theme colors in the submit button also be moved to base button?

Yes that sounds like a good idea!

**Comment by gullwings13** [comment]
Ok cool, we get started on it.

**Comment by gullwings13** [comment]
Ok, I have moved all of that out of submit button and into the base button. I'm not super experienced with Styled Components, so let me know if anything looks out of place.

File: client/src/components/Button/SubmitButton.js [hunk 2]
```diff
@@ -7,48 +7,6 @@ const { button } = theme;
 
 const SubmitButton = styled(BaseButton).attrs(({ size, inline }) => {
```
Comment (anniegiang) [inline client/src/components/Button/SubmitButton.js:8]: Hmm, looks like we don't need SubmitButton anymore. For now I guess we can leave this file as it but we should maybe remove this altogether in the future.

**Pull request merged**